packages/shared/src/components/squads/settings/SquadDangerZone.tsx

```diff
@@ -38,8 +38,8 @@ export function SquadDangerZone({ squad }: SquadDangerZoneProps): ReactElement {
       <DangerZone
         onClick={onDeleteSquad}
         className="mt-4"
-        cta="Delete account"
-        title="Deleting your account will:"
+        cta="Delete squad"
+        title="Deleting your squad will:"
```

Comment:
```suggestion
        title="Deleting your Squad will:"
```

Reply:
In figma this is with lowercase S 🤔

Reply:
Yeah design does it all the time, you can look it up on slack there's been numerous chats.
Squad is a product term, therfor always capital :)

Reply:
Probably was just missed in design, but yeah, this should be on caps as discussed with product waaaaaay back.

Reply:
Just for reference:
https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1699000372453029?thread_ts=1698997971.107649&cid=C05P9ET7S9K